### PR TITLE
docs: document the removal of IA32 Linux support

### PR DESCRIPTION
**Pull request by RaisinTen**
#### Description of Change

Chromium had dropped support for IA32 Linux, so the Chromium
102.0.4999.0 upgrade PR, https://github.com/electron/electron/pull/33731, had introduced the commit,
https://github.com/electron/electron/pull/33731/commits/389ef0731efca802db2089486ad6d9a91cc02ba8, to drop support for IA32 Linux but the change landed
without an addition to the documentation for the breaking changes,
so this PR adds that.

Closes: https://github.com/electron/electron/issues/34783
Refs: https://bugs.chromium.org/p/chromium/issues/detail?id=1194538
Signed-off-by: Darshan Sen <raisinten@gmail.com>

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Review by deepak1556 — CHANGES_REQUESTED** [review]
Electron dropped support ia32 linux a while before chromium dropped it https://www.electronjs.org/blog/linux-32bit-support . The change in the roll is purely internal.

File: docs/breaking-changes.md [hunk 1]
```diff
@@ -98,7 +98,9 @@ is the origin that is checking for device permission.
 
 ## Planned Breaking API Changes (19.0)
 
-None
+### Removed: IA32 Linux binaries
+
+This is a result of Chromium 102.0.4999.0 dropping support for IA32 Linux.
```
Comment (deepak1556) [inline docs/breaking-changes.md:103]: Probably want to link to the `Removed: IA32 Linux support` from here to highlight previous drop in support ?

**Review by deepak1556 — APPROVED** [review]

File: docs/breaking-changes.md (hunk 1 above)
Comment (RaisinTen) [reply docs/breaking-changes.md:103]: Sounds good, done!

File: docs/breaking-changes.md [hunk 2]
```diff
@@ -1265,6 +1268,10 @@ not present, then the native module will fail to load on Windows, with an error
 message like `Cannot find module`. See the [native module
 guide](/docs/tutorial/using-native-node-modules.md) for more.
 
+### Removed: IA32 Linux support
+
+See - [Discontinuing support for 32-bit Linux](https://www.electronjs.org/blog/linux-32bit-support).
```
Comment (VerteDinde) [inline docs/breaking-changes.md:1273]:
```suggestion
Electron 18 will no longer run on 32-bit Linux systems. See [discontinuing support for 32-bit Linux](https://www.electronjs.org/blog/linux-32bit-support) for more information.
```

Sorry to nit, but could we add a very small descriptor here in addition to linking to the blog post (something similar to what you included in the PR comments)? Just thinking of devs scanning the breaking changes sheet when upgrading Electron, wanted to give them a bit more information 🙂

**Review by VerteDinde — APPROVED** [review]

**Pull request merged**